frontend/src/services/http/HttpResponseInterceptors.ts

```diff
@@ -13,10 +14,14 @@ const MAX_RETRY = 2
 // --- Interceptors
 
 // Assume it is caught in error block based on default handling.
-async function forbiddenStatusItcpt(error: any) {
+async function accessErrorResponsesItcpt(error: any) {
+    if (error.response.status == 401) {
+        router.push('/')
+        return Promise.reject(error);
```

Comment:
Hm... sorry Nick, I just noticed something...
Maybe the original 403 retry should have been on **401** because I see from backend "jwt_validation.py" when checking "**ERROR_EXPIRED_TOKEN**" => it throw **401** not 403 (expired token was the reason to retry).
Also, I think in both cases (401- after retry if still fails, and 403 as well) they all need to have `router.push('/')` to home page.
@basilv

Reply:
Yes, I know it would be better to separate the interceptors but I could not find a way to have the interceptors working when they are separated. Maybe it is something I am doing wrong, but I was taking so much time trying to make them work that I decided to keep them together for now.

So, the retry logic should be for 401? I can change it.